src/paper/related_models/paper_model.py

```diff
@@ -776,6 +776,17 @@ def set_paper_completeness(self):
             self.completeness = self.INCOMPLETE
         self.save()
 
+    def download_pdf(self):
+        pass
+
+    def get_pdf_link(self):
```

Comment:
Very small change, but we should have an additional arg `should_save=False|True`